tests/test_other.py

```diff
@@ -416,6 +416,10 @@ def test_multiple_sources(self):
     self.assertNotExists(test_file('twopart_main.o'))
     self.assertNotExists(test_file('twopart_side.o'))
 
+  def test_tsearch(self):
+    self.run_process([EMCC, test_file('tsearch.c'),
+                      ' -s', 'EXIT_RUNTIME=1'])
+    output = self.run_js('a.out.js')
```

Comment:
Replace these twolines with `self.run_other_teast('tsearch.c')`.. then move `tsearch.c` to `tests/other` and create tests/other/test_tseartch.out` alongside it containing some/all of the expected output lines.

I don't think you will need `EXIT_RUNTIME`.